### PR TITLE
Adaptive cards in Teams documentation first draft

### PR DESCRIPTION
**Pull request by hamenon-ms**
Here's the first draft for the upcoming change to add support for adaptive cards. I've kept it fairly simple since a lot of the documentation exists on the adaptive card side already. Do let me know if this is enough or what other changes are required

### CODE REVIEW

**Comment by o365devx** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[msteams-platform/concepts/Cards/cards-actions.md](https://github.com/hamenon-ms/msteams-docs/blob/master/msteams-platform/concepts/Cards/cards-actions.md) | :warning:Warning | | [Details](#user-content-msteams-platform%2Fconcepts%2FCards%2Fcards-actions.md)
[msteams-platform/concepts/Cards/cards-reference.md](https://github.com/hamenon-ms/msteams-docs/blob/master/msteams-platform/concepts/Cards/cards-reference.md) | :warning:Warning | | [Details](#user-content-msteams-platform%2Fconcepts%2FCards%2Fcards-reference.md)
[msteams-platform/assets/images/Cards/adaptivecard.png](https://github.com/hamenon-ms/msteams-docs/blob/master/msteams-platform/assets/images/Cards/adaptivecard.png) | :white_check_mark:Succeeded | |
[msteams-platform/concepts/cards/cards-format.md](https://github.com/hamenon-ms/msteams-docs/blob/master/msteams-platform/concepts/cards/cards-format.md) | :white_check_mark:Succeeded | |

<a name="msteams-platform%2Fconcepts%2FCards%2Fcards-actions.md"></a>
### [msteams-platform/concepts/Cards/cards-actions.md](https://github.com/hamenon-ms/msteams-docs/blob/master/msteams-platform/concepts/Cards/cards-actions.md)
 - **Line 216**: **[Warning]** Invalid file link:(~/concepts/authentication/auth-flow-bot).

<a name="msteams-platform%2Fconcepts%2FCards%2Fcards-reference.md"></a>
### [msteams-platform/concepts/Cards/cards-reference.md](https://github.com/hamenon-ms/msteams-docs/blob/master/msteams-platform/concepts/Cards/cards-reference.md)
 - **Line 41**: **[Warning]** Invalid file link:(~/concepts/cards/cards-actions).
 - **Line 45**: **[Warning]** Invalid file link:(~/concepts/cards/cards-format).
 - **Line 223**: **[Warning]** Invalid file link:(~/concepts/cards/cards-format).
 - **Line 372**: **[Warning]** Invalid file link:(~/concepts/cards/cards-format).
 - **Line 384**: **[Warning]** Invalid file link:(~/get-started/code#microsoft-teams-extensions-for-the-bot-builder-sdk).
 - **Line 470**: **[Warning]** Invalid file link:(~/get-started/code#microsoft-teams-extensions-for-the-bot-builder-sdk).
 - **Line 509**: **[Warning]** Invalid file link:(~/concepts/authentication/auth-flow-bot).
 - **Line 538**: **[Warning]** Invalid file link:(~/concepts/cards/cards-format).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C5%5C23%5Cd98cc23d-94e8-765b-0855-bdd8964dc537%5CPullRequest%5C201805231910212300-157%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=N%2FcUv7F10zCq%2FsDSEFK2BqfsF4IX8g%2FpSgfgXdbPmg4%3D&st=2018-05-23T19%3A06%3A29Z&se=2018-06-23T19%3A11%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by hamenon-ms** [comment]
FYI @billbliss @ninadara @NormSohl

File: msteams-platform/concepts/Cards/cards-reference.md [hunk 1]
```diff
@@ -46,6 +46,159 @@ See the topic [Card Formatting](~/concepts/cards/cards-format) for more informat
...
+
+#### Example
+
+![Example of a adaptive card card](~/assets/images/cards/adaptivecard.png)
+
+#### For more information
+
+[Adaptive Cards Overview](https://docs.microsoft.com/en-us/adaptive-cards/)
+
+*Note: There are some Outlook specific action types like Action.Http which are not supported by Teams. For more information on Outlook specific behavior of adaptive cards please visit: [Designing Outlook Actionable Message cards with the Adaptive Card format](https://docs.microsoft.com/en-us/outlook/actionable-messages/adaptive-card)
```
Comment (aosolis) [inline msteams-platform/concepts/Cards/cards-reference.md]: Might want to put a backslash before the * to escape it. It's fine in the end, but Github is confused and now has the rest of the file italicized.

Comment (aosolis) [inline msteams-platform/concepts/Cards/cards-reference.md]: Should this note be right after the table? It looks a bit off with the 2 intervening sub-sub-sections.

File: msteams-platform/concepts/Cards/cards-reference.md [hunk 2]
```diff
@@ -46,6 +46,159 @@ See the topic [Card Formatting](~/concepts/cards/cards-format) for more informat
...
+
+#### Example
```
Comment (aosolis) [inline msteams-platform/concepts/Cards/cards-reference.md]: This is the second "Example" in this section, which is a bit strange.

File: msteams-platform/concepts/Cards/cards-reference.md [hunk 3]
```diff
@@ -46,6 +46,159 @@ See the topic [Card Formatting](~/concepts/cards/cards-format) for more informat
...
+### Adaptive card
+
+A customizable card that can contain any combination of text, speech, images, buttons, and input fields.
+
+#### Support
+
+| Bots in Teams | Messaging Extensions  | Connectors | Bot Framework |
+| --- | --- | --- | --- |
+| ✔ | ✔ |  ✔* | ✔ |
+|
```
Comment (aosolis) [inline msteams-platform/concepts/Cards/cards-reference.md]: Remove this trailing pipe.

File: msteams-platform/concepts/Cards/cards-reference.md [hunk 4]
```diff
@@ -46,6 +46,159 @@ See the topic [Card Formatting](~/concepts/cards/cards-format) for more informat
 
 ## Cards supported in Teams
 
+### Adaptive card
+
+A customizable card that can contain any combination of text, speech, images, buttons, and input fields.
+
+#### Support
+
+| Bots in Teams | Messaging Extensions  | Connectors | Bot Framework |
```
Comment (aosolis) [inline msteams-platform/concepts/Cards/cards-reference.md]: What does the "Bot Framework" column mean here?

File: msteams-platform/concepts/Cards/cards-reference.md [hunk 5]
```diff
@@ -46,6 +46,159 @@ See the topic [Card Formatting](~/concepts/cards/cards-format) for more informat
...
+| --- | --- | --- | --- |
+| ✔ | ✔ |  ✔* | ✔ |
```
Comment (aosolis) [inline msteams-platform/concepts/Cards/cards-reference.md]: Are all adaptive card actions supported in messaging extensions? Even submit?

File: msteams-platform/concepts/Cards/cards-reference.md (hunk 4 above)
Comment (aosolis) [inline msteams-platform/concepts/Cards/cards-reference.md]: Are adaptive cards not supported in chats? We call out bots in teams, but nothing else.

File: msteams-platform/concepts/Cards/cards-actions.md [hunk 6]
```diff
@@ -214,3 +214,11 @@ var button = new CardAction()
 ## signin
 
 Initiates OAuth flow, allowing bots to connect with secure services, as described in more detail here: [Authentication flow in bots](~/concepts/authentication/auth-flow-bot).
+
+## Adaptive card actions
+
+Adaptive cards support three action types.
+
+* [Action.OpenUrl](http://adaptivecards.io/explorer/Action.OpenUrl.html)
+* [Action.Submit](http://adaptivecards.io/explorer/Action.Submit.html)
```
Comment (aosolis) [inline msteams-platform/concepts/Cards/cards-actions.md]: We need to describe the mechanics of this. For example (not an exhaustive list :))
* what kind of activity is generated?
* what fields are present?
* where do the the card input values go?

**Pull request merged**